README.md

```diff
@@ -57,10 +44,7 @@ runtimes:
 
 Most workloads are sourced from HuggingFace Transformers and are available in
 PyTorch, JAX and Tensorflow. Artifacts are generated from each workload and
-used as input to benchmarks. This decouples the compiler/runtime from the
-framework and enables comparisons across a wider range of runtimes e.g. It is
-possible to run compiler-level comparisons between IREE, XLA and TFLite using
```

Comment:
I think it's fine to leave this in, no? Just describing capabilities of the project.

Reply:
This is still relevant.